apps/hash-api/src/graph/knowledge/primitive/entity.ts

```diff
@@ -523,37 +523,34 @@ export const getEntityIncomingLinks: ImpureGraphFunction<
 /**
  * Get the outgoing links of an entity.
  *
- * @param params.entity - the entity
- * @param params.linkEntityType (optional) - the specific link type of the outgoing links
+ * @param params.entityId - the entityId of the entity to get the outgoing links of
+ * @param [params.linkEntityTypeVersionedUrl] (optional) – the specific link type of the outgoing links to filter to
+ * @param [params.rightEntityId] (optional) – limit returned links to those with the specified right entity
  */
 export const getEntityOutgoingLinks: ImpureGraphFunction<
   {
-    entity: Entity;
-    linkEntityType?: EntityTypeWithMetadata;
-    rightEntity?: Entity;
+    entityId: EntityId;
+    linkEntityTypeVersionedUrl?: VersionedUrl;
+    rightEntityId?: EntityId;
```

Comment:
Only the ids are used in the function, but it was asking for the entire entities. I _think_ this might have led to an entity being needlessly fetched in a caller to satisfy the types, so I've updated to be more precise.